src/frontend/komponenter/Behandling/Personopplysninger/Personopplysninger.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import 'nav-frontend-tabell-style';
+import { useHentPersonopplysninger } from '../../../hooks/useHentPersonopplysninger';
+import { RessursStatus } from '../../../typer/ressurs';
+import InnvandringUtVandring from './InnvandringUtvandring';
+import Barn from './Barn';
+import Adressehistorikk from './Adressehistorikk';
+import Sivilstatus from './Sivilstatus';
+import Fullmakter from './Fullmakter';
+import Statsborgerskap from './Statsborgerskap';
+import { PersonopplysningerHeader } from './PersonopplysningerHeader';
+
+const Personopplysninger: React.FC<{ personIdent: string }> = ({ personIdent }) => {
```

Comment:
Ser att du har brukt `React.FC<{ personIdent: string }>` i stedet for å trekke ut det til en egen Props som det er gjort på mange andre steder. Det kanskje ikke spiller noen rolle? 
Jeg syns selv det føles litt unødvendig å trekke ut i en egen Props når man har 1-2 felter, og kan like gjerne ha inline på alle som du har gjort her.

Reply:
Vad er propsen til Personopplysninger ? Ser legre ned <Personopplysninger id={'1234'} /> mens her er det personIdent  🤔

Reply:
Tenker det bør være personIdent. Endret i komponenten, men glemte å endre i BehandlingContainer.tsx